src/cli/traceViewer/web/ui/networkTab.tsx

```diff
@@ -17,23 +17,18 @@
 import { ActionEntry } from '../../traceModel';
 import './networkTab.css';
 import * as React from 'react';
-import { Expandable } from './helpers';
+import { NetworkResourceDetails } from './networkResourceDetails';
 
 export const NetworkTab: React.FunctionComponent<{
   actionEntry: ActionEntry | undefined,
 }> = ({ actionEntry }) => {
-  const [selected, setSelected] = React.useState(0);
+  const [selected, setSelected] = React.useState('');
```

Comment:
Why don't we make this a number and use index? There could be multiple resources with the same url.

Reply:
I switched it back to being a boolean, and using the mapping index. At some point I wanted to make sure that `expanded` & `selected` values will be reset when switching between actions, as I think it looks a bit weird when you open some network resources, then switch actions, and network resources with the same indices are expanded and one of them is selected.

Anyways, I added an effect to `NetworkResourceDetails` component, to reset those values. It should look better now.